packages/service/lib/schema.js

```diff
@@ -542,30 +542,39 @@ const plugins = {
   }]
 }
 
+const auth = {
```

Comment:
Is it OK to put this in a variable to avoid duplication or do I need to put it in `$defs` and `$ref` it?

Reply:
If the latter, where does it go?
Or is there a better way to express this?